### PR TITLE
Added new help text, fixes #78

### LINKED ISSUE #78: Start/Doc Comparison Page: Simplify instructions
While not everyone read the instructions on how to resolve the errors immediately, those who did completely were able to take the correct steps to remedy the issues. 

Suggestions: 
- Either reduce the amount of content in the help content by eliminating the steps to open VBMS (the user usually has it open already), or 
- Consider hiding these instructions behind an action button right next to the error itself

### PR DESCRIPTION
**Pull request by plusjeff**
image

Did not add help link, since we don't have that content/email address/link/form yet.

### CODE REVIEW

File: app/views/web/start.html.erb [hunk 1]
```diff
@@ -7,8 +7,28 @@
...
+  <div class="instructions">
+    <div class="row">
+      <div class="col-sm-6 col-sm-offset-3">
+        <h3>Find missing documents</h3>
+
+        <p>
+          Caseflow could not find the documents with an <strong>X</strong> below, in the
+          appellant’s eFolder. Try <strong>looking for</strong> the missing document in VBMS and
+          <strong>changing its date</strong> to the date in the VACOLS column. After changing the
+          date, click the <strong>Refresh Page</strong> button below.
```
Comment (alex) [inline app/views/web/start.html.erb]: The button isn't labeled "Refresh Page", either this text or the button's text should be updated

Comment (plusjeff) [inline app/views/web/start.html.erb]: Good catch. Fixed.

**Comment by plusjeff** [comment]
Updated screenshot:

image

**Comment by plusjeff** [comment]
We're going to flip the button and action text orientation.

**Comment by plusjeff** [comment]
Updated screenshot with flipped button and link, column widths changed for top an bottom to match #121  widths/style.

image

**Comment by KHarshawat** [comment]
Looks like the error and missing documents instructions jumped to full width column instead of aligned with the table. Plz fix.

**Comment by alex** [comment]
screen shot 2015-11-05 at 10 19 57 am

**Comment by alex** [comment]
screen shot 2015-11-05 at 10 26 41 am

**Pull request merged**